storage/cql_iface/cql_iface.py

```diff
@@ -0,0 +1,88 @@
+import datetime
+import decimal
+import uuid
+
+import numpy
+
+from storage.cql_iface.tests.mockIStorage import IStorage
+from .cql_comm import CqlCOMM
+from ..storage_iface import StorageIface
+
+"""
+Mockup on how the Cassandra implementation of the interface could work.
+"""
+
+
+class CQLIface(StorageIface):
+    # DataModelID - DataModelDef
+    data_models_cache = {}
+
+    # StorageID - DataModelID
+    object_to_data_model = {}
+    # Object Name - Cache
+    hcache_by_name = {}
+    # Object's class - Cache
+    hcache_by_class = {}
+    # StorageID - Cache
+    hcache_by_id = {}
+
+    # User class to Cassandra data type
+    _hecuba2cassandra_typemap = {
+        None: 'NULL',
```

Comment:
Cassandra can't have a column of type "NULL", NULL is a value not a type. We should remove "None: NULL"